static/js/views/create_cards/CardBuilderBase.js

```diff
@@ -130,10 +130,42 @@ export class CardBuilderBase {
         return description;
     }
 
+    getPowerPoints() {
+        Constants.postData(`${this.baseURL()}/get_power_points`, { card_info: this.cardInfo(), card_id: this.cardID })
```

Comment:
one for the TODO list: use async/await in Javascript. Doesn't have to be now tho.

Reply:
This is using async, but it uses it with then() - why is async/await better?